source/administration/monitoring/publish-events-to-webhook.rst

```diff
@@ -1,4 +1,4 @@
-.. _minio-bucket-notifications-publish-webhook:
+-1;95;0c.. _minio-bucket-notifications-publish-webhook:
```

Comment:
What are these extra characters at the beginning of this line?

Reply:
Whaaaaa? That is bizarre. Fixed.